ca_tools/scripts/control_node.py

```diff
@@ -0,0 +1,185 @@
+#! /usr/bin/env python
+# -*- coding: utf-8 -*-
+#
+# Copyright (c) 2013 PAL Robotics SL.
+# Released under the BSD License.
+#
+# Authors:
+#   * Siegfried-A. Gevatter
+
+import curses
+import math
+from enum import Enum
+import signal
+import sys
+
+import rospy
+from geometry_msgs.msg import Twist
+from geometry_msgs.msg import Pose2D
+from geometry_msgs.msg import Point
+from nav_msgs.msg import Odometry
+import tf
+
+TURNING=1
+STOP_TURNING=2
+FORWARD=3
+STOP_FORWARD=4
+
+def signal_handler(signal, frame,queue_size=1):
+  # your code here
```

Comment:
What is this?

Reply:
Commeted. It's a signal handler to catch the `Ctrl-c` signal